tilt-resources/examples/backupstoragelocations.yaml

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
This is reintroducing files we explicitly removed from the repo, though in a different location.

I wonder if we could generate them at runtime via `velero install -o yaml --dry-run` or something? Maybe as part of the Tiltfile? My worry about having them in the file system is that they'll grow stale as it's a second/third place to maintain things. We also have the `config/samples` directory now, as part of the controller-runtime migration.

Reply:
Good idea. Done.